src/docs/examples/accordion-swe2.html

```diff
@@ -377,6 +377,306 @@ <h4>Related services</h4>
     <div id="qg-one-col" class="row wide">
 
       <div id="qg-primary-content" role="main">
+        <form
+          action="https://www.smartservice.qld.gov.au/services/submissions/email/qld/responsive-maintenance-request"
+          method="post" class="form" data-recaptcha="true">
+
+          <ol class="questions">
+            <li>
+              <label for="name-of-tenant">
+                <span class="label">Name of tenant</span>
+                <abbr class="required" title="(required)">*</abbr>
+              </label>
+              <input id="name-of-tenant" name="name-of-tenant" size="30" required="required"
+                     type="text" />
+            </li>
+
+            <li>
+              <label for="caller-s-name-and-relationship-to-tenant">
+                <span class="label">Callers name and relationship to tenant</span>
+                <small class="hint">If caller is not tenant, fill in relationship details
+                  below</small>
+              </label>
+              <textarea id="caller-s-name-and-relationship-to-tenant"
+                        name="caller-s-name-and-relationship-to-tenant" rows="2" cols="40"></textarea>
+            </li>
+
+            <li>
+              <label for="contact-telephone-numbers">
+                <span class="label">Contact telephone numbers</span>
+                <abbr class="required" title="(required)">*</abbr>
+              </label>
+              <input id="contact-telephone-numbers" name="contact-telephone-numbers" size="20"
+                     type="text" required="required" />
+            </li>
+
+            <li>
+              <label for="rental-agreement-number-or-tenant-s-date-of-birth">
+                <span class="label">Rental agreement number or tenants date of birth</span>
```

Comment:
Needs an apostrophe in `tenant's`.

Reply:
Hi @ThrawnCA 
I have corrected them, these are just example templates to test logic.